### ISSUE TITLE
Dimensions not taking my templating variable correctly

### ISSUE THREAD

**Issue by yzargari** [grafana/grafana#8828]
Hello,

We recently upgraded our ancient version of Grafana to a less ancient one (4.0.2 - we cannot upgrade any further because we're still stuck on Elasticsearch 1.7) and I believe - yet not 100% sure - that this was the point were my problem started.

We are showing graphs based on AWS CloudWatch datasource (metric is us-east-1>AWS/ELB>HealthyHostCount>Stats>Average) and if I enter dimension for LoadBalancerName with exact value it works well.

My dashboard, however is set with templating - so I therefore need to use the templating variable. The name of the ELB is composed of the environment (which is the same for all and not templated) and the cluster number (which is templated). When I set the LoadBalancerName to production-$Cluster - no data points are displayed (as I mentioned, if I enter the exact name it works - e.g. production-database)

I'm pretty sure I'm doing it right, and I can say for sure it did work before. According to the request params sent from my browser to the server - it is only taking the variable value without my prefix:

```
{
  region: "us-east-1",
  action: "GetMetricStatistics",
  parameters: {
    namespace: "AWS/ELB",
    metricName: "HealthyHostCount",
    dimensions: [
      {
        Name: "LoadBalancerName",
        Value: "database"
      }
    ],
    statistics: ["Average"],
    startTime: 1499764104,
    endTime: 1499774904,
    period: 60
  }
}
```

More info:
Grafana version 4.0.2 running on Ubuntu 14.04
Datasource: AWS CloudWatch

Any help is much appreciated!
Thanks,
Yosi

**Comment by torkelo** [comment]
Think this is fixed in Grafana 4.1 or 4.2

**Comment by yzargari** [comment]
Thanks @torkelo
